src/Components/SignUpForm/SignUpForm.test.tsx

```diff
@@ -3,83 +3,62 @@
-
-        wrapper.find('input').first().simulate('change', mockEvent);
-        expect(wrapper.find('button').getDOMNode().disabled).toEqual(true);
-    });
-    it('should render an enabled button if all form fields are filled', () => {
-        const mockEvent: Object = { target: { value: 'Goose' } };
-        const mockEvent2: Object = { target: { value: 'Red' } };
-        const mockEvent3: Object = { target: { value: 'loosegoose@email.com' } };
-        const mockEvent4: Object = { target: { value: 'password123' } };
-        const mockEvent5: Object = { target: { value: 'password123' } };
-
-        const mockToggle = jest.fn();
-        wrapper = mount(<MemoryRouter><SignUpForm
-          isLogin={true}
-          toggleTab={mockToggle}
-          /></MemoryRouter>);
-
-        wrapper.find('input').first().simulate('change', mockEvent);
-        wrapper.find('#last-name').first().simulate('change', mockEvent2);
-        wrapper.find('#email').first().simulate('change', mockEvent3);
-        wrapper.find('#password').first().simulate('change', mockEvent4);
-        wrapper.find('input').last().simulate('change', mockEvent5);
-
-        expect(wrapper.find('button').getDOMNode().disabled).toEqual(false);
-    });
-
-    it('should render an error message if an invalid email is entered', () => {
-      // BUG WITH RENDERING THE ERROR MESSAGE OF THE SIGNUP FORM
-      // DUE TO ASYNC OF SETSTATE THE FORM PROCEEDS TO DASHBOARD BEFORE
-      // UPDATING THE ERROR PORTION OF STATE
-    });
-  });
+  // describe('Change Events', () => {
+  //   it('should render a disabled button if not all form fields are filled', () => {
+  //     const mockEvent: Object = { target: { value: 'Goose' } }
+  //     const wrapper: any = shallow(<SignUpForm {...mockProps} />);
+  //
+  //     wrapper.find('input').first().simulate('change', mockEvent);
+  //     expect(wrapper.find('button').getDOMNode().disabled).toEqual(true);
+  //   });
+  //
+  //   it('should render an enabled button if all form fields are filled', () => {
+  //       const mockEvent: Object = { target: { value: 'Goose' } };
+  //       const mockEvent2: Object = { target: { value: 'Red' } };
+  //       const mockEvent3: Object = { target: { value: 'loosegoose@email.com' } };
+  //       const mockEvent4: Object = { target: { value: 'password123' } };
+  //       const mockEvent5: Object = { target: { value: 'password123' } };
+  //
+  //       const wrapper: any = shallow(<SignUpForm {...mockProps} />);
+  //
+  //       wrapper.find('input').first().simulate('change', mockEvent);
+  //       wrapper.find('#last-name').first().simulate('change', mockEvent2);
+  //       wrapper.find('#email').first().simulate('change', mockEvent3);
+  //       wrapper.find('#password').first().simulate('change', mockEvent4);
+  //       wrapper.find('input').last().simulate('change', mockEvent5);
+  //
+  //       expect(wrapper.find('button').getDOMNode().disabled).toEqual(false);
+  //   });
+  // });
```

Comment:
I found some commented out code when I redid the ActionsField component on my last PR. I am cool with merging this in, but lets maybe take a day to go through and refactor our tests, and make sure every component has at least one test for it